### PR TITLE
Use Cases

### PR DESCRIPTION
**Pull request by timsehn**
This is the first step. I will build a detailed example of each as the second step.

### CODE REVIEW

File: content/SUMMARY.md [hunk 1]
```diff
@@ -17,11 +17,13 @@
...
 - [Use Cases](introduction/use-cases.md)
   - [Data Sharing](introduction/use-cases/data-sharing.md)
-  - [Data Ingestion](introduction/use-cases/data-ingestion.md)
-  - [Machine Learning](introduction/use-cases/ml.md)
-  - [Configuration Management](introduction/use-cases/config.md)
-  - [Version Control Your Application](introduction/use-cases/vc-your-app.md)
-  - [Better MySQL](introduction/use-cases/better-mysql.md)
+  - [Data and Model Quality Control](introduction/use-cases/better-data-and-models.md)
+  - [Manual Data Curation](introduction/use-cases/manual-data-curation.md)
+  - [Version Control for your Application](introduction/use-cases/vc-your-app.md)
```
Comment (zachmu) [inline content/SUMMARY.md:22]: Should the SQL use cases go first?

File: content/introduction/use-cases.md [hunk 2]
```diff
@@ -2,13 +2,23 @@
...
+
+Other customers use Dolt to [manage video game configuration](./use-cases/configuration-management.md), [get an immutable audit log of changes to their database](./use-cases/audit.md), [build reproducibility into machine learning models](./use-cases/data-and-model-quality.md), [ensure data quality using a pull request workflow](./use-cases/manual-data-curation.md), and much more. 
+
+Eventually, we think people will realize that all SQL databases should be version controlled to prevent data loss and make development easier, much the same way we think all code should be version controlled. Dolt compares favorably  on [performance](../reference/sql/benchmarks/latency.md) and [correctness](../reference/sql/benchmarks/correctness.md), as we signalled with [Dolt 1.0](https://www.dolthub.com/blog/2023-05-05-dolt-1-dot-0/).
```
Comment (zachmu) [inline content/introduction/use-cases.md]: This and the following paragraph could be cut

File: content/introduction/use-cases/audit.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+---
+title: Audit
```
Comment (zachmu) [inline content/introduction/use-cases/audit.md:2]: Use cases: Audit

?

File: content/introduction/use-cases/audit.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+# Problem
+
+* Do you need to know who changed what, when, why in your SQL database?
+* Do you want an immutable record of changes going back to the inception of your database?
+* Is an audit team asking for this information for compliance purposes?
+* Do you want to be able to query this audit log like any other tables in your database?
```
Comment (zachmu) [inline content/introduction/use-cases/audit.md]: any other table

File: content/introduction/use-cases/audit.md (hunk 3 above)
Comment (zachmu) [reply content/introduction/use-cases/audit.md:2]: Same for other pages in this section

File: content/introduction/use-cases/configuration-management.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+* Is your configuration too big and complex for files? 
+* Is your configuration more like code than configuration? 
+* Does configuration have a large production impact? 
+* Are configuration changes hard to review? 
+* Are multiple configuration changes hard to merge together when it’s time to ship?
+* Are you building a game with lots of assets and configuration?
+
+# Dolt solves this by…
+
+Configuration is generally structured and managed as large text files. YAML and JSON formatted configuration is very popular. These formats are unordered, meaning standard version control solutions like Git cannot reliably produce [diffs](../../concepts/dolt/git/diff.md) and [merges](../../concepts/dolt/git/merge.md). Moreover, configuration can get quite large running up against the file size limits of tools like Git.
```
Comment (zachmu) [inline content/introduction/use-cases/configuration-management.md]: quite large, running

File: content/introduction/use-cases/data-and-model-quality.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+The version control tools we use to build software apply to modern data science. Version control for data did not exist until Dolt, the first and only database you can [branch](../../concepts/dolt/git/branch.md), [diff](../../concepts/dolt/git/diff.md), and [merge](../../concepts/dolt/git/merge.md) just like a Git repository.
+
+Modern data science applications require model reproducibility, data quality, and multiple versions of data to perform at their best. Dolt allows for these capabilities directly in your database, in a [Git-style version control model](../../concepts/dolt/git/README.md) most developers understand.
+
+The simplest application of Dolt is to use for model reproducibility. If you build a model from a version of the data, make a tag at that commit and refer to that tag in the model metadata. Some of our data and model quality control customers only use Dolt for this simple feature. Dolt shares storage between versions so you can store many more copies of the data using Dolt than say storing copies of the data in S3.
```
Comment (zachmu) [inline content/introduction/use-cases/data-and-model-quality.md]: "is to use for"

File: content/introduction/use-cases/data-and-model-quality.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Dolt allows for human or automated review on data changes increasing data quality. If a bad change makes it through review simply [roll the data back to a previous version](https://www.dolthub.com/blog/2022-09-23-dolt-rollback-options/). DoltHub, DoltLab, and the Hosted Dolt Workbench all implement [a Pull Request](../../concepts/dolthub/prs.md) workflow, the standard for human reviewing code changes. Extend that model to your data changes.
+
+Dolt is the only database with [branch](../../concepts/dolt/git/branch.md) and [merge](../../concepts/dolt/git/merge.md) functionality. Branches allow for long running data projects. Want to add an additional feature to a model, use production data, but you don't want the new feature effecting production model build? Make branch and run the project on that branch. Occasionally merge production data into that branch so you can stay in touch with changes there. Companies use Dolt branches to increase the number of parallel data projects by an order of magnitude.
```
Comment (zachmu) [inline content/introduction/use-cases/data-and-model-quality.md]: ", use production data, but..." is a bit jumbled

File: content/introduction/use-cases/data-and-model-quality.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+Lastly, [commits](../../concepts/dolt/git/commits.md), [logs](../../concepts/dolt/git/log.md), and [diffs](../../concepts/dolt/git/diff.md) for can be used for model insights. Did Thursday's model perform better than Tuesday's but had the same model weights? Inspect the data diff to see what changed. Inspect the commit log to see where that new data came from.
```
Comment (zachmu) [inline content/introduction/use-cases/data-and-model-quality.md]: "for can be used"

File: content/introduction/use-cases/data-and-model-quality.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+# Companies Doing This
+
+[Turbine](https://turbine.ai/), [KAPSARC](https://www.kapsarc.org/), [Flock Safety](https://www.flocksafety.com/), [Tome](https://www.tome.com/), [Bosch](https://www.bosch-home.com/), [IMTF](https://imtf.com/)
```
Comment (zachmu) [inline content/introduction/use-cases/data-and-model-quality.md]: Bullet points?

File: content/introduction/use-cases/manual-data-curation.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+DoltHub and DoltLab support [SQL](../../concepts/dolt/sql/README.md), File Upload (CSV), and a spreadsheet editor for data modification. These interfaces are simple enough that non-technical users can make and review data changes. 
+
+Dolt is a MySQL compatible database so exporting the manually created data to production can be as simple as cloning a copy and starting a server for you developers to connect to.
+
+# Companies Doing This
+
+[Annalise](https://usa.annalise.ai/), [Briya](https://briya.com/), [Aktify](https://aktify.com/), [Blonk Sustainability](https://blonksustainability.nl/), [IMTF](https://imtf.com/), [Lumicks](https://lumicks.com/), [Merkle Science](https://www.merklescience.com/)
```
Comment (zachmu) [inline content/introduction/use-cases/manual-data-curation.md:22]: Bullet points?

File: content/introduction/use-cases/manual-data-curation.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+# Problem
+
+* Are you using spreadsheets to curate production data? 
+* Is the process of merging and reviewing everyone’s changes getting out of hand? 
+* Are bad data changes causing production issues? 
+* Would human review of cell-level data changes help?
+
+# Dolt solves this by…
+
+Dolt allows you to treat your spreadsheet like code. DoltHub and DoltLab implement a [Pull Request workflow](../../concepts/dolthub/prs.md) on tables, the standard for human reviewing code changes. Extend that model to your data changes. Make changes on [branches](../../concepts/dolt/git/branch.md) and then have the changes human reviewed. Data diffs are easily consumed by a human reviewer. Add continuous integration tests to data changes. Have dozens or hundreds of changes in flight at one time.
```
Comment (zachmu) [inline content/introduction/use-cases/manual-data-curation.md]: humans reviewing

File: content/introduction/use-cases/manual-data-curation.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+DoltHub and DoltLab support [SQL](../../concepts/dolt/sql/README.md), File Upload (CSV), and a spreadsheet editor for data modification. These interfaces are simple enough that non-technical users can make and review data changes. 
+
+Dolt is a MySQL compatible database so exporting the manually created data to production can be as simple as cloning a copy and starting a server for you developers to connect to.
```
Comment (zachmu) [inline content/introduction/use-cases/manual-data-curation.md]: for your developers

File: content/introduction/use-cases/offline-first.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+With Dolt you write to the database disconnected. You can have a fully functioning offline application that uses the exact same software and models it would use if it were a standard centralized SQL database.
+
+When it is safe to connect to the internet, Dolt computes the difference between what you have and what a peer database has and only sends these differences both ways. This synchronization process is very efficient, effectively allowing you to get the most information possible in and out in the shortest amount of time. Once the synchronization is complete, go back to disconnected. You and the peer now share a synchronized view with complete, auditable edit history.
+
+Conflicting writes are surfaced quickly and an operator or software can take additional action to resolve.
+
+You get all this with the familiar capabilities and performance of a SQL database.
+
+# Companies Doing This
```
Comment (zachmu) [inline content/introduction/use-cases/offline-first.md:23]: Omit?

File: content/introduction/use-cases/data-and-model-quality.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
+---
+title: Data and Model Quality Control
+---
+
+# Problem
+
+* Are you in the business of creating data and models?
```
Comment (max-hoffman) [inline content/introduction/use-cases/data-and-model-quality.md:7]: Just an idea, most of these say Dolt can do Z, Y, Z, but they leave it to the reader to understand why you would use Dolt compared other products that do X, Y, Z. One angle is maybe "this is a list of products a typical company might use to solve this set of problems; here is what the workflow looks like in that world; vs Dolt bundles the whole package, hosted, offline, backups, UX, OLTP/OLAP perf/you sacrifice Y and Z doing it the old way..."

File: content/introduction/use-cases/versioned-replica.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
+---
+title: Versioned MySQL Replica
+---
+
+# Problem
+
+* Is your production MySQL vulnerable to data loss? 
+* If an operator runs a bad query, script, or deployment can your production MySQL can be down for hours or days as you recover data from backups or logs?
+* Are you worried your backups aren't working?
+* Does internal audit want and immutable log of what changes on your MySQL instance?
```
Comment (zachmu) [inline content/introduction/use-cases/versioned-replica.md]: want an

File: content/introduction/use-cases/versioned-replica.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+* Do you want the ability to copy and sync your production MySQL database for analytics, development, or debugging?
+
+# Dolt solves this by…
+
+Because Dolt is [MySQL-compatible](../../reference/sql/benchmarks/correctness.md), you can set Dolt up as [a versioned replica](../getting-started/versioned-mysql-replica.md) of your MySQL primary. Every transaction commit on your primary becomes a [Dolt commit](../../concepts/dolt/git/commits.md) on the Dolt replica. 
+
+On your Dolt replica, you get full, immutable, queryable audit log of every cell in your database. If an auditor wants guarantees that a cell in your database has not been modified, you can use Dolt to prove it. [Diffs](../../concepts/dolt/git/diff.md) can be produced for every transaction.
```
Comment (zachmu) [inline content/introduction/use-cases/versioned-replica.md]: a full

**Review by zachmu — APPROVED** [review]
Mostly typos, lgtm

File: content/introduction/use-cases/data-and-model-quality.md (hunk 14 above)
Comment (timsehn) [reply content/introduction/use-cases/data-and-model-quality.md:7]: This is a good idea. I'll ponder it. Maybe a section called "What Dolt replaces?"

**Pull request merged**